app/Helpers/database/game.php

```diff
@@ -85,88 +84,103 @@ function getGameMetadata(
     $metricsJoin = '';
     $metricsBindings = [];
     if ($metrics) {
-        $cacheKey = "game:$gameID:playercount";
-        $gameMetrics = Cache::get($cacheKey);
-        if ($gameMetrics === null) {
+        if (env('FEATURE_AGGREGATE_QUERIES')) {
```

Comment:
Please move those to `config/feature.php` and access it via `config('feature.aggregate_query')`.
Configs are cached and thus code loses direct access to env variables at runtime.